apps/server/src/modules/files-storage/controller/files-storage.controller.ts

```diff
@@ -35,10 +35,10 @@ export class FilesStorageController {
 		@CurrentUser() currentUser: ICurrentUser
 	): Promise<StreamableFile> {
 		const res = await this.filesStorageUC.download(currentUser.userId, params);
-		// @TODO set headers ?
+		// TODO set headers ?
 		return new StreamableFile(res.data, {
 			type: res.contentType,
-			disposition: `attachment; filename="${params.fileName}"`,
```

Comment:
I think the filename is not necessary if we set "content-disposition: inline" because it's already part of the request url in a GET request.